### DISCUSSION TITLE
Importing Clementine playlists.

### DISCUSSION THREAD

**Discussion by jtappin** [strawberrymusicplayer/strawberry#1234] [Q&A]
Hello,
I'm trying to transfer my playlists from clementine to strawberry, following the recipe in the [Wiki](https://wiki.strawberrymusicplayer.org/wiki/Import_collection_library_and_playlists_from_Clementine). However I encounter errors on the lines like:
```
   UPDATE strawberry.songs SET artist_id = "";
```
```
Parse error: no such column: 
  UPDATE strawberry.songs SET artist_id = ""; 
                            error here ---^
```
And likewise on the 
```
UPDATE strawberry.playlist_items SET artist_id = "";
```
lines later on.

On starting `strawberry` after following the complete recipe, I see tabs for all of the playlists, but all are empty.

Is there anyone here who know enough `sqlite` to advise on how to investigate this?

I'm running Manjaro Linux (Stable), where clementine has been dropped from the repositories (following Arch) and fails to run with a dependency problem so it's going to be trick to try manually exporting playlists.

**Comment by jonaski** [comment]
Double quotes `"` should be single quotes `'`

**Reply by jtappin** [reply]
Partial success. That gets rid of the errors, but the playlists are still empty.

I should add that all of the tracks within the playlists are local files.
